Cargo.lock

```diff
@@ -21682,31 +21699,34 @@ dependencies = [
 
 [[package]]
 name = "time"
-version = "0.3.27"
+version = "0.3.36"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "0bb39ee79a6d8de55f48f2293a830e040392f1c5f16e336bdd1788cd0aadce07"
+checksum = "5dfd88e563464686c916c7e46e623e520ddc6d79fa6641390f2e3fa86e83e885"
 dependencies = [
  "deranged",
  "itoa",
  "libc",
+ "num-conv",
  "num_threads",
+ "powerfmt",
  "serde",
  "time-core",
  "time-macros",
 ]
 
 [[package]]
 name = "time-core"
-version = "0.1.1"
+version = "0.1.2"
```

Comment:
Wait why do we need this ? I would say a stable release should use the same toolchain as we used for the initial version.

Reply:
Time does not build anymore with newer compilers, but this does not fix it since it is already releases.  
Maybe revert and just emerge, CI wont be happy anytime soon.

Reply:
> Time does not build anymore with newer compilers

I see, ideally we would want to use the same compiler images between same stable minor releases.

Reply:
@ggwpez reverted.